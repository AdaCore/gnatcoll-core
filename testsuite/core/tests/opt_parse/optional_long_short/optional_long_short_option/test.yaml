description: Check that custom usage text can be provided for Opt_Parse Options
run_args:
    - ["-C1", "latin-1", "-C2", "latin-2", "--charset3", "latin-3"]
    - ["--help"]
control:
    - [SKIP, "env.is_cross", "Tests using test.sh currently not supported on cross targets, see T616-039"]
