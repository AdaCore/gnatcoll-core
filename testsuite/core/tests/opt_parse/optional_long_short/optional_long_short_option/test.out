stdout contained expected output for INPUT1
stdout contained expected output for INPUT2
stdout contained expected output for INPUT3